### ISSUE TITLE
NVDA STOPS WORKING IN WINDOWS EXPLORER ON WINDOWS 10

### ISSUE THREAD

**Issue by netblue44** [nvaccess/nvda#8164]
### Steps to reproduce:
OPEN WINDOWS EXPLORER THRU ANY MEANS > A list of the steps you take to demonstrate the problem. 
>NVDA JUST MAKES A EROR sound and does not read anything  
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.
NVDA should work normally 
### Actual behavior:
> Tell us what happens instead.

### System configuration:
Windows 10 pro
nvda_snapshot_next-15023,fcbd1731

**Comment by PratikP1** [comment]
This is verified. It’s a duplicate of #8161though.

**Comment by leonardder** [comment]
Closing as duplicate indeed. I just pushed a fix for this to @dkager's branch.

**Comment by leonardder** [comment]
Closing as duplicate indeed. I just pushed a fix for this to @dkager's branch.
